TheSteelGuy/Ride-My-WayApiV1
# [chore]  add failing testcases for the endpoints

**Introduction**
This pr creates test cases for the various api endpoints
**Description of completed tasks**

- created tests for various use cases

**How to test this manually**

- Ensure you have python 2.7 installed
- make a virtual environment
- Activate your environment
- Clone the repo
- cd to the repo
- run pytest --cov-report term-missing --cov=api

cd to the cloned directory
